composer.json

```diff
@@ -21,9 +21,9 @@
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
     "oat-sa/extension-tao-ltideliveryprovider" : ">=12.0.0",
     "oat-sa/generis" : ">=14.0.0",
-    "oat-sa/tao-core" : ">=47.0.0",
+    "oat-sa/tao-core" : ">=48.39.0",
     "oat-sa/extension-tao-delivery" : ">=15.0.0",
-    "oat-sa/extension-tao-lti" : ">=13.2.0",
+    "oat-sa/extension-tao-lti" : "dev-feature/TR-1919/LtiSubmissionReviewRequest as 100",
```

Comment:
Please do not forgot to change it to proper version before merging

Reply:
done